pdr_backend/subgraph/prediction.py

```diff
@@ -101,6 +105,7 @@ def mock_daily_predictions() -> List[Prediction]:
         "binance",
         0.0,
         1701503100,
+        "0x18f54cc21b7a2fdd011bea06bba7801b280e3151",
```

Comment:
we can make this value as a constant in the same file.

Reply:
Consider this mock/test data, and not to be abstracted/cleaned. Preferably, this would be a csv or json file.